requirements/development.in

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+-r base.in
+-e file:.[cors,druid,hive,mysql,postgres,thumbnails]
+flask-cors>=2.0.0
```

Comment:
This is a bit of an hack to force `pip-compile-multi` to generate the necessary extra dependency locks. Since `-e file:.[cors,druid,hive,mysql,postgres,thumbnails]` was not being propagated to `development.txt`

Reply:
All I see in `development.txt` is `-e file:.`. The fact that you're explicitly enumerating these requirements in `development.in` should be more than suffice to generate the desired outcome in `development.txt`.

Reply:
right, updated